src/components/Whiteboard/ElementBehaviors/Resizable/ResizeElement.test.tsx

```diff
@@ -157,4 +168,57 @@ describe('<ResizeElement />', () => {
       type: 'path',
     });
   });
+
+  it('should resize line elements', () => {
+    render(<ResizeElement elementId="element-1" />, {
+      wrapper: Wrapper,
+    });
+
+    // drag a resize box from 0,0 to 50,50
```

Comment:
`// drag a resize box from 0,0 to 50,50` does in my opinion not fit in this case.
This test should drag the start of the line instead.